backend/src/sms/services/sns-sms-client.class.ts

```diff
@@ -0,0 +1,33 @@
+import { SNSClient, PublishCommand, PublishInput } from '@aws-sdk/client-sns'
+import config from '@core/config'
+
+const region = config.get('aws.awsRegion')
+
+export default class SnsSmsClient {
+  private client: SNSClient
+  private senderId: string
+
+  constructor(senderId = 'postman') {
```

Comment:
I'm not super sure whether we should default to Postman. will it seem like phishing if messages from say MOM suddenly started coming from Postman?

Reply:
Maybe we can have a fallback on the UI also to allow them to enter a sender ID.

Reply:
As discussed, we'll default to Gov.sg